test/clientSpec.js

```diff
@@ -33,7 +33,7 @@ const info = {
 
 const peerAddress = 'example.red.server'
 const options = {
-  server: 'btp+wss://user:placeholder@example.com/rpc'
+  server: 'btp+wss://user:shhh secret@example.com/rpc'
```

Comment:
actually, since this is a URI, I think this should be `shhh%20secret`, then?

Reply:
When passing the connection string to the [URL constructor](https://github.com/interledgerjs/ilp-plugin-payment-channel-framework/blob/9cf6c18f10a803f4dfef3629498113f361f03771/src/lib/client.js#L18) the string is encoded. So this string here should not be url-encoded. Otherwise, it would get encoded twice.

Reply:
what if the username contains a `:` or the password contains a `@`?

Reply:
It can't. I guess that's the downside of cramming the credentials into the URL. Wasn't there the option to define username and password as separate properties? (cc @sharafian )

Reply:
There was originally an option to specify credentials separately but it was removed in favor of a single BTP URI format